package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "description": "Proxy server project",
   "main": "index.js",
-  "scripts": {  
-    "start": "node ./training/nodejs/js/index.js"
+  "scripts": {
+    "start": "node --env-file .env ./training/nodejs/js/index.js"
```

Comment:
looks like i would be better to move entrypoint (index.js) to root directory and then just run 'node index.js' - dotenv will resolve .env file

Reply:
Moved index.js, but I left "--env-file .env" because do not use dotenv module